### PR TITLE
update the package metadata for new pypi site

### PR DESCRIPTION
**Pull request by willingc**
Update the metadata which is displayed on pypi.

Do we want to update the email address?

Reminder: Update twine and setuptools prior to cutting the release as recent versions are needed for long description metadata.

### CODE REVIEW

**Comment by MSeal** [comment]
Feel free to merge if you have an updated email address or if you want to put what's there in for releasing today that's fine too.

File: setup.py [hunk 1]
```diff
@@ -55,17 +66,20 @@ def read(fname):
...
     cmdclass=versioneer.get_cmdclass(),
-    description='Parametrize and Run Jupyter Notebooks',
+    description='Parametrize and run Jupyter and nteract Notebooks',
     author='nteract contributors',
     author_email='jupyter@googlegroups.com',
     license='BSD',
-    keywords="jupyter mapreduce",
-    long_description=(open('README.rst').read() if exists('README.rst') else ''),
+    # Note that this is a string of words separated by whitespace, not a list.
+    keywords='jupyter mapreduce nteract pipeline notebook',
```
Comment (rgbkrk) [inline setup.py]: Ooooh love this pipeline word.

**Comment by rgbkrk** [comment]
We don't have an nteract address at the moment, though we certainly could. 😄

**Comment by MSeal** [comment]
@rgbkrk Should we make the author_email our group email for papermill (since we're an open sorta group)?

**Comment by rgbkrk** [comment]
Yeah.

**Comment by willingc** [comment]
Merged author_email in #194

**Pull request merged**